module_3/task3.js

```diff
@@ -0,0 +1,51 @@
+import csvtojson from "csvtojson"
+import fs from "fs"
+const CSV_FILE_PATH = ".csv/data-csv.csv"
+const TXT_FILE_PATH = ".csv/data-txt.txt"
+
+const onError = (err) => {
+  console.error(err)
+  throw new Error(err)
+}
+
+const processedJson = (json) => {
+  const fullItems = []
+  const byLine = json.split("\r")
+  const header = byLine[0].toLowerCase().split(",")
+  header.splice(2, 1)
+  for (let i = 1; i < byLine.length; i++) {
+    const line = byLine[i].substring(1).split(",")
+    line.splice(2, 1)
+    const parsed = {}
+    for (const index in line) {
+      parsed[header[index]] = line[index]
+    }
+    fullItems.push(parsed)
+  }
+  return fullItems
+}
```

Comment:
the idea of csvtojson is avoiding you to make this manual work. So check these links and see how you can do it:

* streaming data: https://www.npmjs.com/package/csvtojson#asynchronously-process-each-line-from-csv-url
* IgnoreColumns: https://www.npmjs.com/package/csvtojson#api

Reply:
I didn't know you could mix subscribe with the pipes. The links don't work, but I get it from the comment itself

Thank you!